frontend/apps/reader/modules/readerlink.lua

```diff
@@ -917,16 +924,21 @@ function ReaderLink:onGoToPageLink(ges, internal_links_only, max_distance)
         end
         if shortest_dist then
             selected_distance2 = shortest_dist
+            if max_distance and selected_distance2 > max_distance^2 then
+                selected_link = nil
+            end
         end
     else
-        -- Getting segments on a page with many internal links is
-        -- a bit expensive. With larger_tap_area_to_follow_links=true,
-        -- this is done for each tap on screen (changing pages, showing
-        -- menu...). We might want to cache these links per page (and
-        -- clear that cache when page layout change).
-        -- If we care only about internal links, request them only
-        -- (and avoid that expensive segments work on external links)
+        -- Getting segments on a page with many internal links is a bit expensive.
+        -- With larger_tap_area_to_follow_links=true,
+        -- this is done for each tap on screen (changing pages, showing menu...).
```

Comment:
You have no idea how much time it took me to get these 3 lines comment's text nearly justified (carefully picking words, reading synonyms dictionaries...) - and look how trashy ragged right you made them ! :)

Reply:
I was wondering about that since it seemed so far below the 80col limit :D.

Reply:
This stills hurt my eyes (if would not if first line ended with a period, but it's a comma there):
```lua
        -- With larger_tap_area_to_follow_links=true,
        -- this is done for each tap on screen (changing pages, showing menu...).
```

Reply:
Oh, crappy GH inline comments strike again, didn't see that one. Reflowed it just now ;).